app/models/loader/types.rb

```diff
@@ -134,8 +134,24 @@ def verify; end
     end
 
     class Host < Record
-      def verify; end
-      def_delegators :@policy_object, :restricted_to
+      def_delegators :@policy_object, :restricted_to, :apikey
```

Comment:
It's not clear what adding `:apikey` is accomplishing here. Can you elaborate on that?

Reply:
It is no longer needed. I deleted it